### PR TITLE
v1.2.22

### PR DESCRIPTION
**Pull request by flarco**
# 1.2.22

1. **Direct Insert Support**
- Added new `SLING_DIRECT_INSERT` environment variable to enable direct table inserts
- Implemented `writeToDbDirectly()` function to bypass temp table creation
- Added new test cases for direct insert functionality

2. **File Path Improvements**
- Added `CleanTableName()` function to sanitize table names in temp file paths
- Updated temp file paths across multiple database connectors to use cleaned table names
- Made temp file paths more consistent and descriptive

3. **Stream Configuration Updates**
- Refactored stream configuration handling to use structured configs instead of maps
- Added `LoaderStreamConfig()` for standardized loader configurations
- Improved compression and datetime format handling

4. **Database Connector Enhancements**
- Added trusted connection support for SQL Server
- Updated Prometheus client authentication
- Improved error handling and transaction management
- Enhanced column casing support for upserts

### CODE REVIEW

**Comment by flarco** [comment]
@yokofly can you give an eye to https://github.com/slingdata-io/sling-cli/pull/410/commits/b0f82b16ea0de182091dfa408465465677304111? Anything you think is missing?

I am going to release tomorrow to address some DuckDB bug I found.

**Comment by yokofly** [comment]
@flarco ok, I will check today later.

File: core/sling/task_run_write.go [hunk 1]
```diff
@@ -343,6 +350,141 @@ func (t *TaskExecution) WriteToDb(cfg *Config, df *iop.Dataflow, tgtConn databas
...
+	}
+
+	// Create final table
+	if err := createTable(t, tgtConn, targetTable, sampleData, false); err != nil {
+		return 0, err
+	}
+
+	cfg.Target.Columns = sampleData.Columns
+	df.Columns = sampleData.Columns
+	setStage("5 - load-into-final")
```
Comment (yokofly) [inline core/sling/task_run_write.go:402]: I check the old writetodb use `cfg.Target.Columns = sampleData.Columns` seems not the old logic:
https://github.com/slingdata-io/sling-cli/blob/73216e1946a28d46fec2a9c7d840bb851349f5b3/core/sling/task_run_write.go#L191-L193
```
	cfg.Target.Options.TableDDL = g.String(tableTmp.DDL)
	cfg.Target.TmpTableCreated = true
	df.Columns = sampleData.Columns
	setStage("4 - load-into-temp")

```

Comment (yokofly) [reply core/sling/task_run_write.go:402]: @flarco plz check the above logic if related^

Comment (flarco) [reply core/sling/task_run_write.go]: Ok, I removed the `cfg.Target.Columns = sampleData.Columns` line. It actually didn't really impact anything.

**Pull request merged**